src/pages/Book/BookedPlace.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+import variables from '../../styles/variables';
+
+const dateToString = date => {
+  const dateArray = date.split('-');
+  return `${dateArray[0]}년 ${dateArray[1]}월 ${dateArray[2]}일`;
+};
+
+const BookedPlace = ({ bookList }) => {
+  return (
+    <S.BookListWrapper>
+      {bookList &&
+        bookList.map((list, idx) => (
```

Comment:
```suggestion
        bookList.map((bookInfo, idx) => (
```
map callback 함수의 파라미터는 단일 값이기 때문에 단수형으로 적어주면 더 좋을 것 같습니다!

Reply:
그리고 list 에서 나오는 값 콘솔에 찍어서 한번 보여주실 수 있나요?
데이터 형태 보고 싶어서 그렇습니다.

Reply:
<img width="1120" alt="Screen Shot 2022-11-08 at 2 41 02 PM" src="https://user-images.githubusercontent.com/70960594/200485695-632892cc-b7f7-4f48-a4e5-b33d04600a8c.png">